ios/Classes/SwiftDeltaChatCorePlugin.swift

```diff
@@ -172,12 +170,6 @@ public class SwiftDeltaChatCorePlugin: NSObject, FlutterPlugin {
         result(UIApplication.version)
     }
 
-    fileprivate func setCoreStrings(methodCall: FlutterMethodCall, result: FlutterResult) {
-        if let coreStrings: CoreStrings.CoreStringsDictionary = methodCall.arguments as? CoreStrings.CoreStringsDictionary {
-            CoreStrings.sharedInstance.strings = coreStrings
-        }
-        result(nil)
-    }
```

Comment:
So, shouldn't we get rid of the `CoreStrings` class, too?

Reply:
Definitely yes. Didn't saw it while navigating through the iOS project.